src/main/java/org/kairosdb/core/http/rest/QueryAutocompleter.java

```diff
@@ -0,0 +1,49 @@
+package org.kairosdb.core.http.rest;
+
+import com.google.common.collect.SetMultimap;
+import org.kairosdb.core.datastore.QueryMetric;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class QueryAutocompleter {
+    private static final Logger logger = LoggerFactory.getLogger(QueryAutocompleter.class);
+
+    public void complete(QueryMetric query) {
+        completeMetricTag(query);
```

Comment:
I'm not sure I'm fully understand the reason behind extracting method `completeMetricTag`, instead of inlining it?

Reply:
The idea is that in future there might be other similar autocompletions added, and then one public method will look like:
```java
    public void complete(QueryMetric query) {
        completeMetricTag(query);
        completeAnotherTag(query);
        completeThridTag(query);
```